### DISCUSSION TITLE
Nested relations sorting not working

### DISCUSSION THREAD

**Discussion by vishalinfyom** [rappasoft/laravel-livewire-tables#724] [Q&A]
We have nested relations and we are trying to add sorting on it using callback function, but its not working.

Example. i have following tables ::

## appointments
 - patient_id (primary key of patients table)
 
## Patients
- id
- user_id (primary of users table)

## Users
- id
- first_name

Now i'm listing appointments by using sorting on user's first name. So the sorting will be looks like

```....
->('patient.user', function (Builder $q) {
 $q->where('first_name', 'desc')
))

**Comment by xitude** [comment]
Can you supply what version you are using?

**Reply by vishalinfyom** [reply]
v2.3.0
